File: services/Titular.php
<?php

class Titular
{
    private string $nome;
    private string $cpf;
    private string $endereco;

    public function __construct(string $nome, string $cpf, string $endereco)
    {
        $this->setNome($nome);
        $this->setCpf($cpf);
        $this->setEndereco($endereco);
    }

    public function setNome(string $nome): void
    {
        $trimmedNome = trim($nome);
        if (strlen($trimmedNome) < 3) {
            throw new Exception("O nome do titular deve ter pelo menos 3 caracteres.");
        }

        if (preg_match('/\d/', $trimmedNome)) {
            throw new Exception("O nome do titular não pode conter números.");
        }

        $this->nome = $trimmedNome;
    }

    public function setCpf(string $cpf): void
    {
        if (strlen($cpf) !== 11 || !ctype_digit($cpf)) {
            throw new Exception("O CPF deve conter exatamente 11 dígitos numéricos.");
        }

        $this->cpf = $cpf;
    }

    public function setEndereco(string $endereco): void
    {
        $trimmedEndereco = trim($endereco);
        if ($trimmedEndereco === "") {
            throw new Exception("O endereço não pode ser vazio.");
        }

        $this->endereco = $trimmedEndereco;
    }

    public function getNome(): string
    {
        return $this->nome;
    }

    public function getCpf(): string
    {
        return $this->cpf;
    }

    public function getEndereco(): string
    {
        return $this->endereco;
    }

    public function __toString(): string
    {
        return sprintf(
            "Nome: %s, CPF: %s, Endereço: %s",
            $this->nome,
            $this->cpf,
            $this->endereco
        );
    }
}

?>

File: model/Titular.cs
using System;

namespace SistemaBancario
{
    public class Titular
    {
        private string nome;
        private string cpf;
        private string endereco;

        public Titular(string nome, string cpf, string endereco)
        {
            setNome(nome);
            setCpf(cpf);
            setEndereco(endereco);
        }
        
        public Titular()
        {
        }

        public void setNome(string nome)
        {
            if (nome.Trim().Length < 3)
            {
                throw new ArgumentException("O nome do titular deve ter pelo menos 3 caracteres.");
            }

            if (nome.Any(char.IsDigit))
            {
                throw new ArgumentException("O nome do titular não pode conter números.");
            }

            this.nome = nome;
        }

        public void setCpf(string cpf)
        {
            if (cpf.Length != 11 || !cpf.All(char.IsDigit))
            {
                throw new ArgumentException("O CPF deve conter exatamente 11 dígitos numéricos.");
            }

            this.cpf = cpf;
        }

        public void setEndereco(string endereco)
        {
            if (string.IsNullOrWhiteSpace(endereco))
            {
                throw new ArgumentException("O endereço não pode ser vazio.");
            }

            this.endereco = endereco;
        }

        public string getNome()
        {
            return nome;
        }
        public string getCpf()
        {
            return cpf;
        }
        public string getEndereco()
        {
            return endereco;
        }

        public override string ToString()
        {
            return $"Nome: {nome}, CPF: {cpf}, Endereço: {endereco}";
        }
    }
}